fixtures/room/room_fixtures.py

```diff
@@ -23,3 +23,56 @@
         }
     }
 }
+
+
+rooms_query = '''
+    {
+    rooms{
+        edges{
+            node{
+                name
+                capacity
+                roomType
+                }
+            }
+        }
+    }
+    '''
+query_rooms_response = {
+    "data": {
+        "rooms": {
+        "edges": [
+            {
+            "node": {
+                "name": "Entebbe",
+                "capacity": 6,
+                "roomType": "meeting"
+            }
+            }
+            
+        ]
+        }
+    }
+    }
+
+room_query_by_id = '''
+                {
+                getRoomById(id:1){
+                    capacity
+                    name
+                    roomType
+                }
+                }
+                '''
+
+room_query_by_id_response = {
```

Comment:
This response is only valid if the fields in the payload are exactly as this, remember user can request for any set of fields in the payload, you might need more fixtures to reflect the fields users might want in the response data

Reply:
This has been resolved